### ISSUE TITLE
can't see errors from angular 

### ISSUE THREAD

**Issue by pocesar** [systemjs/systemjs#1374]
I'm using JSPM, but it's a problem with SystemJS.

I'm doing a refactor on my code to use System output from Typescript, and so far, I'm trying to blindly trying to fix errors that are appearing because SystemJS seems to be swallowing the error message from angular.

![systemjs](https://cloud.githubusercontent.com/assets/461084/17082370/1436e1ac-5152-11e6-9531-1e0fec6020ad.png)

load.js is:

```js
(function(){
    var paths = ["/javascripts/checker/main.js","/javascripts/app/main.js"];

    function importNext () {
        var path = paths.shift();
        if (path) {
            System.import(path).then(importNext, function(err){
                console.warn(path);
                console.log(err);
                importNext();
            })
        } else {
            console.log('loaded');
        }
    }

    importNext();
})()
```

How can I see the original exception?

**Comment by pocesar** [comment]
Ok, I couldn't find anywhere in the docs about originalErr, so I'm just leaving this here to help people having the same issue:

```js
System.import(path).then(importNext, function(err){
                console.warn(path);
                console.log(err.originalErr);
                importNext();
            })
```

**Comment by guybedford** [comment]
What specific error is this? If you can share a replication it would be really useful to understand how SystemJS is obscuring the original message and if there is a better way to pick up the original message in https://github.com/ModuleLoader/es6-module-loader/blob/master/src/wrapper-start.js#L41 to avoid it.

**Comment by pocesar** [comment]
alright:
- System.import('app/main')
- app/main.js calls `import * as Base from './modules/base';`
- modules/base.js calls `import * as angular from 'angular'`
- modules/base.js tries to load a non-yet-loaded angular module, through `angular.module('Base', ['NotYet'])`
- it fails "silently", it shows the minErr, but with no message from angular.

**Comment by guybedford** [comment]
@pocesar I've completely forgotten how to use Angular here, so can you perhaps suggest how to make the following code give an error that covers your use case? I just tried the following:

```javascript
    SystemJS.import('angular').then(function(angular) {
      angular.module('myModule', ['not-existing']).
      config(function(injectables) {
      }).
      run(function(injectables) {
      });
      console.log('done?');
    });
```

but get no errors.

**Comment by pocesar** [comment]
@guybedford my files are SystemJS modules with anonymous register calls (generated from Typescript). You'd have to wrap the code before trying haha

```js
System.register(['angular', '../modules/base'], function(exports_1, context_1) {
    'use strict';
    var __moduleName = context_1 && context_1.id;
    var angular;
    return {
        setters:[
            function (angular_1) {
                angular = angular_1;
            },
            function (_1) {}],
        execute: function() {
            angular
                .module('Main', [
                'Base',
                'Client'
            ]);
            angular.bootstrap(document, ['Main']);
        }
    }
});
```

**Comment by guybedford** [comment]
Thanks for clarifying. Fixed in https://github.com/ModuleLoader/es6-module-loader/commit/07aa5671f4719f6625886ae14c9fde995b06af8a.

**Comment by guybedford** [comment]
Released in 0.19.34.
